apps/src/sites/code.org/pages/public/teacher-dashboard/sections.js

```diff
@@ -33,9 +31,6 @@ export function renderSectionsPage(data) {
   store.dispatch(setStudioUrl(data.studiourlprefix));
   store.dispatch(setValidLoginTypes(data.valid_login_types));
   store.dispatch(setValidGrades(data.valid_grades));
-  if (!experiments.isEnabled('sectionFocus')) {
-    store.dispatch(setValidAssignments(data.valid_courses, data.valid_scripts));
```

Comment:
Are `valid_courses` still used elsewhere?  It looks like this might have been the last usage (`SectionsPage` asks the dashboard API for this). If so, maybe it can be removed starting with teacher-dashboard/index.haml and the JSDoc in this file.